Library/Homebrew/test/Gemfile

```diff
@@ -1,9 +1,13 @@
 source "https://rubygems.org"
 
-gem "mocha", "~> 1.1"
-gem "minitest", "~> 5.3"
-gem "rake", "~> 10.3"
-gem "parallel_tests", "~> 2.9"
+gem "minitest"
```

Comment:
We can remove this as well, since `minitest` has been part of the standard library since 1.9.3.